core/contracts/composed_stream_template.kf

```diff
@@ -629,6 +637,10 @@ procedure insert_metadata(
         VALUES ($uuid, $key, $value_i, $value_f, $value_s, $value_b, LOWER($value_ref), $current_block);
 }
 
+procedure insert_default_base_date($base_date text) public {
+    insert_metadata('default_base_date', $base_date, 'string');
+}
```

Comment:
insert_metadata is already a public procedure, I don't think it is needed to make another procedure on top of it unless there is some logic being used.